bot/exts/tags.py

```diff
@@ -70,6 +70,37 @@ async def suggest_friends(self, interaction: AppCmdInter) -> None:
 
         await interaction.followup.send(embed=embed, ephemeral=True)
 
+    @tag.sub_command()
+    async def info(self, interaction: AppCmdInter, member: Member) -> None:
+        """View tags and info of given member."""
+
+        tag_repo = self.bot.tag_repository
+        if not tag_repo:
+            raise DatabaseNotConnectedError
+        tag_list = await tag_repo.get(member.id)
+        name = member.name
+        if member.nick:
+            name = f"{member.nick} ({name})"
+        # [1:] to remove the @everyone in the roles list
+        info_dict = {
+            "**Member Info**": " ",
+            "Joined:": f"<t:{int(member.joined_at.timestamp())}:R>",
+            "Roles:": ", ".join(role.mention for role in member.roles[1:]),
+            "Tags:": ", ".join(f"`{tag}`" for tag in tag_list),
```

Comment:
add a check for no tags to display that instead of blank

```py
", ".join(f"`{tag}`" for tag in tag_list) if len(tag_list) > 0 else "No Tags"
``` 
or something to the same effect

Reply:
blank checked at line 95 👍